### DISCUSSION TITLE
A difference between the results of the rheoFoam and rheoHeatFoam

### DISCUSSION THREAD

**Discussion by Joe199999** [fppimenta/rheoTool#135] [General]
Hi guys
I solved a steady problem using the rheoFoam and got a good convergence. Then continued that using the rheoHeatFoam (by addition of related parameters like T, ...). But rheoHeatFoam changed the results of hydrodynamic parameters, like velocity, pressure, drag coef, etc about 2%.
No viscous dissipation and the properties are T-independent.
You can also check this using the tutorial problem named cylinder (of90/tutorials/rheoHeatFoam/cylinder/powerLaw). 
Do you know why?

**Comment by YAlperenDegirmenci** [comment]
Hi Joe,

Thanks for sharing your experience with this. I'm currently working on my own case and have observed similar behavior. After switching from rheoFoam to rheoHeatFoam (without viscous dissipation and with temperature-independent properties), the hydrodynamic parameters (velocity, pressure, etc.) show a deviation, even with good convergence in both solvers.

I was wondering if you ever found the reason behind this discrepancy?

**Reply by Joe199999** [reply]
Hi
Unfortunately, I haven't found the reason.
